lib/ansible/modules/crypto/openssl_certificate.py

```diff
@@ -0,0 +1,760 @@
+#!/usr/bin/python
```

Comment:
https://www.python.org/dev/peps/pep-0394/ would be `#!/usr/bin/env python`, but I haven't looked up conventions in the Ansible code base.